### ISSUE TITLE
[TF2] [Bug] Scoreboard -> Spectate player not working

### ISSUE THREAD

**Issue by Ploppy3** [ValveSoftware/Source-1-Games#2190]
In casual at least, you can't spectate your teammates anymore via the scoreboard.

**Comment by kisak-valve** [comment]
Possible duplicate of #2086 / #2115.

**Comment by Ploppy3** [comment]
@kisak-valve It's another issue, even though the menu is extremely hard to reach due to the issues you pointed out, clicking on spectate does nothing at all.

**Comment by agrastiOs** [comment]
Also possibly related to #2034.

**Comment by Ploppy3** [comment]
@agrastiOs I told 2 times already that it's the button that is not working, spectating works fine.

**Comment by agrastiOs** [comment]
Yes, but it might be related. Relating doesn't mean that it's the same thing.

**Comment by Ploppy3** [comment]
Oh my mad, never thought about it that way. Thank you.

**Comment by agrastiOs** [comment]
Well, it's OK. Everyone's new to something :)
